pkg/auth/rights/hook.go

```diff
@@ -55,7 +55,9 @@ func Hook(next grpc.UnaryHandler) grpc.UnaryHandler {
 		if !ok {
 			panic(errNoFetcher)
 		}
-		ids, ok := req.(ttnpb.Identifiers)
+		ids, ok := req.(interface{
+			CombinedIdentifiers() *ttnpb.CombinedIdentifiers
+		})
```

Comment:
Can't use `ttnpb.CombinedIdentifiers` because of import cycle?

Reply:
Assuming you meant `ttnpb.Identifiers`: no, because request messages only need to implement the func `CombinedIdentifiers() *ttnpb.CombinedIdentifiers` for this to work, while `ttnpb.Identifiers` also contains a bunch of other things that we shouldn't want to implement on request messages.

Reply:
Nah I overlooked, I mean the one in `pkg/events` but we certainly don't want that dep here